applications/GeoMechanicsApplication/custom_conditions/README.md

```diff
@@ -0,0 +1,141 @@
+
+Microclimate boundary conditions are determined by factors such as:
+
+- Air temperature
+- Solar radiation
+- Humidity
+- Precipitation
+- Wind speed
+
+## Radiation
+The net radiation $R_n$ $\mathrm{[W/m^2]}$ is calculated by:
+
+$$ R_n = R^s + R^{l,ab} - R^{l,em} $$
+
+where
+
+- $R^s$ = in-coming short wave radiation $\mathrm{[W/m^2]}$
+- $R^{l,ab}$ = absorbed long wave radiation $\mathrm{[W/m^2]}$
+- $R^{l,em}$ = emitted long wave radiation $\mathrm{[W/m^2]}$
+
+The incoming short wave radiation is defined by
+
+$$ R^s = \left(1 - \alpha \right) R_{g} $$
+
+where
+
+- $\alpha$ = Albedo cover or vegetation coefficient  $\mathrm{[-]}$
+- $R_{g}$ = Short wave radiation  $\mathrm{[W/m^2]}$ 
+
+$R_g$ is a user difined parameter which is usually provided as a "Time-Radiation" table.
+
+The absorbed long wave radiation is:
+
+$$ R^{l,ab}=  \epsilon \sigma \left( T_{at} +273.15 \right)^4 $$
+
+where
+
+- $\epsilon$ = effective emissivity  $\mathrm{[-]}$
+- $\sigma$ = Stefan Boltzmann's constant  $\mathrm{[5.67 \cdot 10^{-8}  W/m^{2 \circ}C^4]}$
+- $T_{at}$ = atmospheric temperature $\mathrm{[^{\circ}C]}$
+
+And the emitted long wave radiation,
+
+$$ R^{l,em} = \sigma \left( T_{ss} +273.15 \right)^4 $$
+
+where $T_{ss}$ $\mathrm{[^{\circ}C]}$ is land surface temperature.
+
+## Surface Heat Storage
+The surface heat storage term $Q_s$ $\mathrm{[W/m^2]}$ is calculated by,
+
+$$ Q_s =  a_1 R_n(t+\Delta t) + a_2 \frac{R_n(t+\Delta t) - R_n(t)}{\Delta t} + a_3 $$
+
+where $a_1$ $\mathrm{[-]}$, $a_2$ $\mathrm{[s]}$ and $a_3$ $\mathrm{[W/m^2]}$ are user defined variables.
+
+
+## Evaporation Flux
+The evaporation flux $E$ $\mathrm{[m/s]}$ is calculated by,
+
+$$ E = \frac{L_v}{\lambda \rho_w} $$
+
+where
+
+- $L_v$ = latent heat flux
+- $\lambda$ = latent heat of vaporization $\mathrm{[2.45 \space MJ/kg]}$
+- $\rho_w$  = density of water $\mathrm{[kg/m^3]}$
+
+The latent heat flux is:
+
+$$ L_v  = \frac{e^\prime_{at} \left(R_n + Q_f - Q_s \right) + C_a \rho_a \left(e_{at}^s - e_{at}^a\right) / r_a}{e^\prime_{at} + \gamma (1+r_{s} / r_a)} $$
+
+where
+
+- $r_a$ = atomospheric resistance $\mathrm{[s/m]}$
+- $r_s$ = surface resistance constant $\mathrm{[30 \space s/m]}$ 
+- $\gamma$ = psychometric constant $\mathrm{[0.67 \space hPa/ ^{\circ}C]}$
+- $Q_f$ = build enviromental radiation $\mathrm{[W/m^2]}$ (user defined parameter)
+- $C_a$ = specific heat of moist air $\mathrm{[J/kg ^{\circ}C]}$
+- $\rho_a$ = air density $\mathrm{[kg/m^3]}$ 
+- $e^s$ = saturated vapour pressure $\mathrm{[hPa]}$ 
+- $e^\prime$ = slope of the saturation vapor curve $\mathrm{[hPa/ ^{\circ}C]}$
+- $e^a$ = actual vapour pressure $\mathrm{[hPa]}$
+
+$$ e^s = 6.11 \exp \left( \frac{17.27 \space T}{T+237.3} \right) $$
```

Comment:
parameters from what source? Buck (1981) for saturation over water gives different values.